web/help/GuideUnit1.md

```diff
@@ -625,9 +651,9 @@ Coloring
 --------
 
 The first transformation you will use is coloring.  The `colored` function
-changes the color of a picture. This function expects two arguments: the
-preimage, and a new color.  The colors of the preimage don't matter at
-all; only the shapes involved.  The result of the `colored` function is a new
+changes the color of a picture. This function expects two arguments: 1) the
+preimage and 2) a new color.  The colors of the preimage don't matter at
+all, only the shapes involved.  The result of the `colored` function is a new
```

Comment:
I don't think either a comma or semicolon is correct here, actually.  A dash should work.